redaxo/src/addons/debug/boot.php

```diff
@@ -114,6 +114,36 @@
 
     $ep->table('Executed Extension Points', rex_extension_debug::getExtensionPoints());
     $ep->table('Registered Extensions', rex_extension_debug::getExtensions());
-
-    $clockwork->resolveRequest()->storeRequest();
-});
+};
+
+$console = rex::getConsole();
+if ($console) {
+    $console->setShutdownFunction(function (\Symfony\Component\Console\Event\ConsoleTerminateEvent $event) use ($shutdownFn, $console){
+        $shutdownFn();
+
+        $command = $event->getCommand();
+        $input = $event->getInput();
+        $output = $event->getOutput();
+        $exitCode = $event->getExitCode();
+
+        $clockwork = rex_debug_clockwork::getInstance();
+        $clockwork
+            ->resolveAsCommand(
+                $command->getName(),
+                $exitCode,
+                $input->getArguments(),
+                $input->getOptions(),
+                $command->getDefinition()->getArgumentDefaults(),
+                $command->getDefinition()->getOptionDefaults()
+                // $output->fetch()
```

Comment:
hier erstmal bewusst ohne den output aus der console... da kann ggf. sehr viel inhalt in der console sein, daher würde ich das hier nicht "blind" in die website rendern

Reply:
aktuell haben wir IMM keinen Command der ne sehr große Menge an Output gibt. Ich würds aktuell eher mit reinnehmen.

Reply:
dazu muss man die console so einstellen dass der output gepuffert wird etc.

würd ich in einem separaten PR machen, wenn wir das wollen